nginx-default.conf

```diff
@@ -5,5 +5,17 @@ server {
         root /usr/share/nginx/html;
         index index.html index.htm;
         try_files $uri /index.html;
+
+        # Add security headers
+        add_header X-Content-Type-Options nosniff;
+        add_header X-Frame-Options SAMEORIGIN;
+        add_header X-XSS-Protection "1; mode=block";
+        add_header Referrer-Policy no-referrer-when-downgrade;
+        add_header Content-Security-Policy "default-src 'self'; script-src 'self' https://www.google-analytics.com https://www.googletagmanager.com; style-src 'self' https://fonts.googleapis.com; font-src 'self' https://fonts.gstatic.com; connect-src 'self' *.hasadna.org.il;";
+
+        # CORS settings
+        add_header Access-Control-Allow-Origin "self https://www.google-analytics.com https://www.googletagmanager.com https://fonts.gstatic.com https://fonts.googleapis.com *.hasadna.org.il";
+        add_header Access-Control-Allow-Methods "GET, POST, OPTIONS";
+        add_header Access-Control-Allow-Headers "Origin, X-Requested-With, Content-Type, Accept";
```

Comment:
Thank you!
But regarding CORS settings - is it needed? I was under the impression that CORS is blocked by default for all origins, and that google analytics and google fonts doesn't need to be included here.